bindings/swift/tree_sitter/parser.h

```diff
@@ -0,0 +1 @@
+src/tree_sitter/parser.h
```

Comment:
it feels off to have `bindings/swift/tree_sitter` contain the `parser.h` for the `tree-sitter-pkl` parser, when there exists a `bindings/swift/TreeSitterPkl`. Shouldn't `parser.h` simply be included from `pkl.h`?

Reply:
This was temporary, I think there is actually no need for parser.h at all to be exposed to Swift